src/app/Page/main-view/main-view.component.ts

```diff
@@ -30,22 +45,29 @@ export class MainViewComponent implements OnInit {
       .subscribe({
         next: (employees) => {
 
-          this.employees = employees  // *taking the value of responce and populating it into a variable.
+          // Dean you will want to split these into the collections defined above based on the value of status
+          this.employees[0].employees = employees  // *taking the value of responce and populating it into a variable.
 
           console.log(employees);
         }
       })
 
   }
 
-  drop(event: CdkDragDrop<string, Employee, Employe>) {
+  drop(event: CdkDragDrop<EmployeeStatus, EmployeeStatus, Employee>) {
```

Comment:
this is the type,

if is container a, container b, type of the thing. if that makes sense